docs/src/gettingstarted/introduction/template/cli-install.md

```diff
@@ -12,65 +12,36 @@ With all of the requirements met, install the CLI to start developing with Platf
 
 {{< asciinema src="videos/asciinema/verify-cli-extended.cast" >}}
 
-1. **Install the CLI**
+## 1. Install the CLI
 
-    In your terminal, run the following command depending on your operating system:
+{{% cli-installation %}}
 
-{{< codetabs >}}
+## 2. Test the CLI
 
----
-title=Linux/macOS
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -fsS https://platform.sh/cli/installer | php
-
-<--->
-
----
-title=Windows
-file=none
-highlight=bash
-markdownify=false
----
-
-curl -f https://platform.sh/cli/installer -o cli-installer.php
-php cli-installer.php
-
-{{< /codetabs >}}
-
-
-2. **Authenticate and Verify**
-
-   Once the installation has completed, you can run the CLI in your terminal with the command
-
-   ```bash
-   platform
-   ```
-
-   {{< note >}}
+To interact with your projects you need to be logged in.
+To open the authentication page and get a list of your Platform.sh projects, run:
 
-   If you opened your free trial account using another login (such as GitHub),
-   you can't authenticate with this command until you setup your account password with Platform.sh in the console.
+```bash
+platform
+```
 
-   {{< /note >}}
+## 3. Clone your project locally
 
-   You should now see a list of your Platform.sh projects, including the template you made in this guide.
-   You can copy its *project ID* hash, and then download a local copy of the repository with this command:
+To clone your project locally:
 
-   ```bash
-   platform get <project ID>
-   ```
+1. Get a list of your Platform.sh projects with: `platform project:list`.
```

Comment:
:lady_beetle: Given that the last action of the previous step returned a list of projects, this seems slightly redundant.